whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -17304,31 +17329,51 @@
     },
     "462": {"ignored": true, "NOTE:record-count": 0},
     "480": {
-      "addLink": "marc:hasSeeFromTracingGeneralSubdivision",
-      "resourceType": "marc:SeeFromTracingGeneralSubdivision",
-      "$i": {"addProperty": "marc:referenceInstructionPhrase"},
-      "$v": {"addProperty": "marc:formSubdivision"},
-      "$w": {"property": "marc:controlSubfield"},
-      "$x": {"addProperty": "marc:generalSubdivision"},
-      "$y": {"addProperty": "marc:chronologicalSubdivision"},
-      "$z": {"addProperty": "marc:geographicSubdivision"},
-      "$4": {"ignored": true},
-      "$5": {"addProperty": "marc:institutionToWhichFieldApplies"}
+      "addLink": "hasVariant",
+      "resourceType": "TopicSubdivision",
+      "pendingResources": {
+        "_:topicSubdivision": {
+          "addLink": "hasVariant",
+          "resourceType": "TopicSubdivision",
+          "required": true,
+          "itemPos": "first",
+          "absorbSingle": true
+        }
+      },
```

Comment:
From what I can see, pendingResource isn't needed in this case (since it's not a nested entity).

Reply:
Fixed it! :)

Reply:
\o/